newrelic/newrelic.yml

```diff
@@ -0,0 +1,300 @@
+# This file configures the New Relic Agent.  New Relic monitors
+# Java applications with deep visibility and low overhead.  For more details and additional
+# configuration options visit https://docs.newrelic.com/docs/java/java-agent-configuration.
+#
+# This configuration file is custom generated for app73246343@heroku.com
+#
+# This section is for settings common to all environments.
+# Do not add anything above this next line.
+common: &default_settings
+
+  # ============================== LICENSE KEY ===============================
+  # You must specify the license key associated with your New Relic
+  # account. For example, if your license key is 12345 use this:
+  # license_key: '12345'
+  # The key binds your Agent's data to your account in the New Relic service.
+  license_key: '9f76c3b721b65bcb9934c8a1ad3e54d01b1d1a0d'
```

Comment:
Not sure there is any way around this, but do we need to have the key be publicly exposed in this file? Would it be possible to make this an environment variable we read in instead?